### ISSUE TITLE
Support for nativescript-vue-template-compiler

### ISSUE THREAD

**Issue by cainrus** [TypeStrong/fork-ts-checker-webpack-plugin#354]
## Feature motivation
Compatibility with nativescript-vue. It uses nativescript-vue-template-compiler instead of vue-template-compiler

## Feature description
Allow to set ```{ vue: {compiler: 'nativescript-vue-template-compiler'}}``` instead of ```vue: true```
To be compatible with nativescript-vue based projects.

## Feature implementation
We need to fix types for vue option from ```boolean``` to ```boolean | { compiler: string }```
Pass vue options to VueProgram.ts if they are available, then update ```require('vue-template-compiler')``` and error message in catch block.

**Comment by johnnyreilly** [comment]
If you'd like to work a PR we'd certainly take a look!
